.build/external-documentation/cli-command-reference.quickstart.yml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: docs-cli-command-reference
   annotations:
     order: 4
+  labels:
+    service-category/application-services: "true"
+    product/rhosak: "true"
+    product/rhosr: "true"    
+    product/rhoad: "true"
```

Comment:
Remove the rhoad entry -- CLI is not yet supported.

Reply:
sorry missed this because display name different to file name: 
displayName: Installing and configuring the rhoas CLI
I was updating 3 CLI docs so missed the removal of this.

Reply:
No problem -- this doc wasn't included in the original commit so I hadn't reviewed it earlier. (Btw, it's the "CLI command reference (rhoas)", not "Installing and configuring the rhoas CLI", but both CLI guides apply to the same services.)